schedule/yast/opensuse/agama_autoinst.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: agama_auto
+description: >
+  First test for autoinstallation wit agama
+vars:
+  AGAMA_AUTO: yam/agama/auto/alp.sh
```

Comment:
please move this to job group, so the same schedule could be reuse with different scripts.